streamwebs_frontend/streamwebs/templates/streamwebs/navigation.html

```diff
@@ -0,0 +1,20 @@
+<div id="topbar">
+    <div class="navbar">
+        <div class="navbar right"
+             align="right"></div>
```

Comment:
I am wondering if you should remove the alignment because Alex might try to style it in css and there might be some conflicts... you should probably ask @subnomo